projects/fsastorefrontlib/src/cms-components/seller-dashboard/seller-dashboard.component.html

```diff
@@ -0,0 +1,103 @@
+<ng-container *ngIf="seller$ | async as user; else loading">
+  <div class="row">
+    <div class="col-md-5 mb-5 mb-md-0">
+      <div class="box-shadow border border-color-3">
+        <h5 class="text-center py-3 mb-0">
+          {{ 'dashboard.yourProfile' | cxTranslate }}
+        </h5>
+        <div class="user-profile-wrapper background-color-4">
+          <div class="user-profile bg-white border border-color-3 text-center">
+            <span class="user-image h1 fas fa-user p-3"></span>
+            <h6 class="text-uppercase">
+              {{ 'dashboard.name' | cxTranslate }}
+            </h6>
+            <h6 class="notice">{{ user.name }}</h6>
+            <hr />
+            <ul class="text-left mb-0">
+              <li class="d-flex justify-content-between align-items-end">
+                <span class="semi-bold pr-2"
+                  >{{ 'dashboard.email' | cxTranslate }}:</span
+                >
+                <span class="notice">
+                  {{ user.uid }}
+                </span>
+              </li>
+              <ng-container *ngIf="user.contactInfos?.length">
+                <li class="d-flex justify-content-between align-items-end">
+                  <span class="semi-bold pr-2"
+                    >{{ 'fscommon.phoneNumber' | cxTranslate }}:</span
+                  >
+                  <span class="notice">
+                    {{ user.contactInfos[0]?.phoneNumber }}
+                  </span>
+                </li>
+              </ng-container>
+              <li class="d-flex justify-content-between align-items-end">
+                <span class="semi-bold pr-2"
+                  >{{
+                    'fscommon.register.dateOfBirth.label' | cxTranslate
+                  }}:</span
+                >
+                <span class="notice">
+                  {{ user.dateOfBirth | parseDate | date: 'dd MMM yyyy' }}
+                </span>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-7">
+      <ng-container *ngIf="customers$ | async as customers; else loading">
+        <div class="box-shadow border border-color-3">
+          <h5 class="text-center py-3 mb-0">
+            {{ 'dashboard.overview' | cxTranslate }}
+          </h5>
+          <div class="dashboard-overview-wrapper background-color-4">
+            <ul
+              class="dashboard-overview-stats mb-0 bg-white border border-color-3 p-3"
+            >
+              <li class="item text-break text-center text-uppercase">
+                <div class="number notice semi-bold">
+                  {{
+                    customers?.entries?.length > 0
```

Comment:
Also should be refactored with more generic approach in this [task](https://sapjira.wdf.sap.corp/browse/CXFSA-412)